WeatherInfo/ViewControllers/Locations View Controller/Add Location View Controller/AddLocationViewController.swift

```diff
@@ -38,19 +38,21 @@ class AddLocationViewController: UIViewController {
         viewModel.locationsDidChange = { [unowned self] (locations) in
             self.tableView.reloadData()
         }
-        
+        self.activityIndicatorView.isHidden = true
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
         viewModel.queryinhDidChange = { [unowned self] (querying) in
             if querying {
+                self.activityIndicatorView.isHidden = false
                 self.activityIndicatorView.startAnimating()
             } else {
+                self.activityIndicatorView.isHidden = true
                 self.activityIndicatorView.stopAnimating()
             }
         }
-    }
-
-    override func viewDidAppear(_ animated: Bool) {
```

Comment:
Why did you remove viewDidAppear

Reply:
I would want searchBar to become the first responder when the view is being presented.

Reply:
Yes it is searchBar is made first responder still in viewDidAppear. I did not remove it.